learning_resources/etl/prolearn.py

```diff
@@ -29,6 +29,10 @@
 # List of query fields for prolearn, deduced from its website api calls
 PROLEARN_QUERY_FIELDS = "title\nnid\nurl\ncertificate_name\ncourse_application_url\ncourse_link\nfield_course_or_program\nstart_value\nend_value\ndepartment\ndepartment_url\nbody\nbody_override\nfield_time_commitment\nfield_duration\nfeatured_image_url\nfield_featured_video\nfield_non_degree_credits\nfield_price\nfield_related_courses_programs\nrelated_courses_programs_title\nfield_time_commitment\nucc_hot_topic\nucc_name\nucc_tid\napplication_process\napplication_process_override\nformat_name\nimage_override_url\nvideo_override_url\nfield_new_course_program\nfield_tooltip"  # noqa: E501
 
+SEE_EXCLUSION = (
+    '{operator: "<>", name: "department", value: "MIT Sloan Executive Education"}'
+)
+
 # Performs the query made on https://prolearn.mit.edu/graphql, with a filter for program or course  # noqa: E501
 PROLEARN_QUERY = """
```

Comment:
Minor suggestion: If we add another `%s`, depending on placement, breaks current usage. Maybe could do

```python
def prolearn_query(course_or_program: str, sloan_filter: bool, fields: str) -> str:
    return f"""
<query as f-string>
"""
```